network.tf

```diff
@@ -1,13 +1,13 @@
-locals {
-  allowed_ips = ["38.34.109.158/32","10.0.0.0/28","146.115.59.209/32"]
+variable "allowed_ips"{
+  default = ["38.34.109.158/32","10.0.0.0/28","146.115.59.209/32"]
 }
 
 resource "aws_vpc" "vpc" {
     cidr_block = "10.0.1.0/24"
     enable_dns_support   = true
     enable_dns_hostnames = true
     tags       = {
-        Name = "FAH EKS VPC"
+        Name = "() EKS VPC"
```

Comment:
Do the parentheses get dynamically populated with the project name? If not, why are they here?

Reply:
no? this was supposed to be interpolated